docs/source/chaincode4ade.rst

```diff
@@ -106,16 +107,22 @@ fabric contract api package and define our SmartContract.
 
 Next, let's add a struct ``Asset`` to represent simple assets on the ledger.
 Note the JSON annotations, which will be used to marshal the asset to JSON which is stored on the ledger.
+JSON though is not a deterministic data format - the order of elements can change, whilst still representing the same data semantically.
+The challenge, therefore, is to be able to generate a consistent set of JSON. Below is also shown a good approach to use to achieve consistency.
+The latter consists of creating the asset object struct following alphabetic order
```

Comment:
The last two sentences are a little confusing... you mention "below" and "latter". Are you referencing a single example or two different examples? If a single example, then maybe combine the two sentences into one.